### DISCUSSION TITLE
Make a way to favorite Plugins/Addons from the asset lib

### DISCUSSION THREAD

**Discussion by Bluzinhoj** [godotengine/godot-proposals#8893] [General Discussions]
There are so many assets and interesting tools made by the community that its easy to lose track of some cool assets, demos or whatever.

Wouldn't be cool if we have something like a favorites folder(inside the editor) with the plugins we choose to put there?

And to go even further, imagine if u could group your favorites plugins at will, using labels, categories or whatever. To seprate them at any way like "usefull plugins for topdown shooters" "usefull for rpgs" and etc... all of that from the editor and the asset lib page.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/3521.
